tests/raftstore/test_compact_log.rs

```diff
@@ -14,21 +14,30 @@
 use std::collections::HashMap;
 
 use tikv::raftstore::store::*;
+use tikv::storage::CF_RAFT;
+use tikv::util::rocksdb::get_cf_handle;
+use rocksdb::DB;
+use protobuf;
 use kvproto::raft_serverpb::RaftApplyState;
 
 use super::util::*;
 use super::cluster::{Cluster, Simulator};
 use super::node::new_node_cluster;
 use super::server::new_server_cluster;
 
+fn must_get_msg_cf<M>(engine: &DB, cf: &str, key: &[u8]) -> M
+    where M: protobuf::Message + protobuf::MessageStatic
+{
+    engine.get_msg_cf(cf, key).unwrap().unwrap_or_default()
```

Comment:
IMO, must here means the key must exist, but unwrap_or_default means key maybe not exist.